.azure-pipelines/MZoo-CI-Windows.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+- master
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'vs2017-win2016'
```

Comment:
can you try "windows-2019" ? as this is the latest and includes VS 2019... it will matter when we start using ort for validating the model... in the ort CIs I believe we use the 2019 image... doesnt mean this will not work just nice to be consistent

Reply:
Upgraded to Windows 2019.